src/App.jsx

```diff
@@ -1,35 +1,28 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useEffect, useState } from "react";
+import Nav from "./Components/Nav";
+import { Routes, Route } from "react-router-dom";
+import axios from "axios";
+import ListOfArticles from "./Components/ListOfArticles";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://nc-news-by-amir.onrender.com/api/articles")
```

Comment:
Great use of axios, you should separate out your API calls into functions in it's own `api.js` file, then import them where needed around your app.
This way you can just import axios into one file as well as separating out your concerns 👍 
Should this API call happen here or in the `ListOfArticles`? 🤔